### DISCUSSION TITLE
umi下从antdv4升级到antdv5出现 not found /antd/dist/antd.less

### DISCUSSION THREAD

**Discussion by xiaogonggong-w** [umijs/umi#10548] [Q&A]
umi版本:3.5.0
antd版本: 5.2.1
<img width="842" alt="image" src="https://user-images.githubusercontent.com/79799040/219941875-dd8f0d4e-cdaf-47c5-b6f6-27fb7742d90b.png">
现在antd5不是采用的是cssinjs么,所以在antd里面应该没有less文件,但是我项目里面也没有引入这个文件,目前只在.umi/umi.ts中出现这个.
<img width="931" alt="image" src="https://user-images.githubusercontent.com/79799040/219941899-fcaf8277-a0db-40c6-989e-1ae90ba93dc6.png">
所以,广大的朋友们,有谁遇到过这个问题么,应该怎么解决

**Comment by fz6m** [comment]
umi 3 的 antd 插件还没适配 antd 5 ，现在还用不了。
